docusaurus/docs/extensions/advanced/fix-annotations-published-extensions.md

```diff
@@ -0,0 +1,20 @@
+# Fixing annotations on published extensions
+
+As the development of an extension progresses over time, through different versions of Rancher and the Shell package, looking back at those earlier releases some may think why is that extension version available for install in Rancher versions when you know it shouldn't work.
+
+This is due to the fact that when the first releases were done, no-one could foresee if future versions would have breaking changes, therefore [annotations](../extensions-configuration#configurable-annotations) most commonly used like `catalog.cattle.io/rancher-version` or `catalog.cattle.io/kube-version` would either lack an upper limit boundary or just have too low of an upper limit.
+
+Since that extension was already built and served to users, fixing annotations and repackaging an extension my be a pain point that would often get postponed.
+
+With Shell `3.0.0` we have introduced a mechanism that will allow developers to fix annotations on those published extension version that will handle changes on `index.yaml` that will persist over time and also the re-packaging of the `tgz` assets with the correct annotations inside. To leverage this mechanism, developers should follow this procedure:
+
+1) After identifying which extension version you want to fix the annotations for, go to branch of the repository where the Helm chart that serves your extension lives and edit the `Chart.yaml` in `charts/<-YOUR-EXT->/<-EXT-VERSION-TO-FIX->/Chart.yaml` and change the annotations there as desired
+
+2) Once the annotations changes are done, on the root folder of the skeleton app that contains the extension just run the following command:
+```js
+yarn publish-pkgs -e -g
+```
```

Comment:
Just for clarification, does this mean pushing changes to the branch where the helm charts are served, then going back to the main branch to run the `publish-pkgs` script to see those changes?

Reply:
Your question raised me some concerns after reading this... There's no package.json on gh-pages branch... 🤔 I think I'll have to tweak this

Reply:
But to answer your question, you can't change branches otherwise it won't work... But I do get the point where your comment came from